Templates/functional-template/src/config.ts

```diff
@@ -1,26 +1,16 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
+// config settings, env variables
 import path from 'path';
 import { config as dotenv } from 'dotenv';
+import { IConfig } from './interfaces';
 
 // Load .env file into process.env if it exists. This is convenient for running locally.
 dotenv({
-  path: path.resolve(__dirname, '../../.env'),
+  path: path.resolve(__dirname, '../../.env.template'),
```

Comment:
the template is there for you create your own .env file, so this will read your own .env file if it exists, or it will go with the k8s variables if it doesnt (production)

Reply:
yeah, that's right. fixed ✅